e2e/core/global-setup.ts

```diff
@@ -0,0 +1,29 @@
+import { request } from '@playwright/test';
+
+/**
+ * This configuration is to reuse the signed-in state in the tests
+ * by log in only once using the API and then skip the log in step for all the tests.
+ *
+ * https://playwright.dev/docs/auth#reuse-signed-in-state
+ */
+
+async function globalSetup() {
```

Comment:
Is there a reason not to use the API fixture to do this?

Reply:
The scope of a fixture is limited to a spec. There is an ongoing discussion here regarding implementing a global scope: https://github.com/microsoft/playwright/issues/14895#issuecomment-1205498922